packages/javascript/bh-shared-ui/src/components/AppIcon/Icons/utils.tsx

```diff
@@ -30,8 +30,9 @@ export const BaseSVG: React.FC<
     }
 > = (props) => {
     const { size = 16, name, children, ...rest } = props;
+
     return (
-        <svg {...rest} width={size} height={size}>
+        <svg width={size} height={size} {...rest}>
```

Comment:
by overriding width/height with square dimensions throws off icons that are not square. To ensure this change didnt introduce regressions, I removed any previous width/height definitions on BaseSVG (that wouldve been overrode anyway but came from figma) so that size definitions would still fall thru. I then confirmed all uses of those icons still looked the same and were sized correctly.